docs.config.js

```diff
@@ -7,7 +7,7 @@ const docs = [
   },
 ];
 
-if (process.env && process.env.HEAD !== 'production') {
+if (process.env && process.env.CONTEXT !== 'production') {
```

Comment:
Where is this env variable set out of interest?

Reply:
https://www.netlify.com/docs/continuous-deployment/#environment-variables

It seems really hard to test that these do what you want without spinning up a new site/repo to test deployments on.